apps/notifications/src/panel/templates/undo-list-item.jsx

```diff
@@ -90,15 +90,15 @@ export class UndoListItem extends React.Component {
 	};
 
 	spamComment = () => {
-		var comment = wpcom()
+		const comment = wpcom()
 			.site( this.props.note.meta.ids.site )
 			.comment( this.props.note.meta.ids.comment );
-		var component = this;
+		const component = this;
```

Comment:
This looks redundant. There's not even a candidate for arrow function that would use the variable. Just one simple statement that could be converted to `this.finishExecution()`